Added basic choose file method for #9:

easyaspi314 opened the pull request:
Export function adds the date/time (do we really need a custom name?)

Import has a dialog that lists all the files in /sdcard/HW-Manager and clicking it imports it. 

I've removed the file extension using Apache IO's FilenameUtils (a trimmed version using only what we need), and the NOTICE file is the Apache 2.0 license for that file.

easyaspi314 committed:
Added basic choose file method for #9:

Export function adds the date/time (do we really need a custom name?)

Import has a dialog that lists all the files in /sdcard/HW-Manager and clicking it imports it. 

I've removed the file extension using Apache IO's FilenameUtils (a trimmed version using only what we need), and the NOTICE file is the Apache 2.0 license for that file.

AltNico commented:
Very nice, I opened the issue just 12 hours before your pull request came. Thank you for your work!

I thought of something like in [Flym](https://github.com/FredJul/Flym), take a look [here](https://github.com/FredJul/Flym/blob/master/app/src/main/java/net/fred/feedex/fragment/EditFeedsListFragment.java#L440), this maybe would be easier...

easyaspi314 commented:
You’re welcome! 

> On Jan 24, 2015, at 5:21 AM, Nico Alt notifications@github.com wrote:
> 
> Very nice, I opened the issue just 12 hours before your pull request came. Thank you for your work!
> 
> I thought of something like in Flym https://github.com/FredJul/Flym, take a look here https://github.com/FredJul/Flym/blob/master/app/src/main/java/net/fred/feedex/fragment/EditFeedsListFragment.java#L440, this maybe would be easier...
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/hw-manager/android/pull/12#issuecomment-71309403.

easyaspi314 committed:
Fixed import function.
Suppressed some Lint errors.
Changes a few minor things.
About Constants was broken, so I fixed it. You do not need the CDATA tag, otherwise, it showed the HTML as it looked like in a plain text editor.

AltNico committed:
reformat code using AS

AltNico committed:
fix date stamp
